integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/datamap/CGDataMapTestCase.scala

```diff
@@ -420,6 +419,63 @@ class CGDataMapTestCase extends QueryTest with BeforeAndAfterAll {
       sql("select * from normal_test where name='n502670' and city='c2670'"))
   }
 
+  test("test cg datamap and validate the visible and invisible status of datamap ") {
```

Comment:
The above testcase can be removed.

Reply:
Ok, I will add some validate in old test case

Reply:
ok, I removed。 and change old test case